apps/web/src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue

```diff
@@ -1,104 +1,70 @@
-import DashboardWidgetContainer from '@/services/dashboards/shared/dashboard-widget-container/DashboardWidgetContainer.vue';
-import DashboardLabels from '@/services/dashboards/shared/DashboardLabels.vue';
-import DashboardRefreshDropdown from '@/services/dashboards/shared/DashboardRefreshDropdown.vue';
 import { useDashboardDetailInfoStore } from '@/services/dashboards/store/dashboard-detail-info';
 
+
 interface Props {
     dashboardId?: string;
 }
 
 const props = defineProps<Props>();
+
 const dashboardDetailStore = useDashboardDetailInfoStore();
 const dashboardDetailState = dashboardDetailStore.$state;
 
 const state = reactive({
-    name: dashboardDetailState.name,
-    apiParam: computed<Partial<DashboardConfig>>(() => ({
-        name: dashboardDetailState.name,
-        labels: dashboardDetailState.labels,
-        settings: dashboardDetailState.settings,
-        layouts: [dashboardDetailState.dashboardWidgetInfoList],
-        variables: dashboardDetailState.variables,
-        variables_schema: dashboardDetailState.variablesSchema,
-        tags: { created_by: store.state.user.userId },
-    })),
     loading: false,
 });
 
-/* Api */
-const getDashboardData = async () => {
-    try {
-        await dashboardDetailStore.getDashboardInfo(props.dashboardId);
-    } catch (e) {
-        ErrorHandler.handleError(e);
-        await SpaceRouter.router.push({ name: DASHBOARDS_ROUTE.ALL._NAME });
-    }
+const handleClickBackButton = () => {
+    SpaceRouter.router.back();
 };
+
 const updateDashboardData = async () => {
     try {
+        state.loading = true;
+
         if (dashboardDetailStore.isProjectDashboard) {
             await SpaceConnector.clientV2.dashboard.projectDashboard.update({
-                ...state.apiParam,
-                name: state.name,
+                name: dashboardDetailState.name,
+                labels: dashboardDetailState.labels,
+                settings: dashboardDetailState.settings,
+                layouts: [dashboardDetailState.dashboardWidgetInfoList],
+                variables: dashboardDetailState.variables,
+                variables_schema: dashboardDetailState.variablesSchema,
+                tags: { created_by: store.state.user.userId },
                 project_dashboard_id: props.dashboardId,
             });
         } else {
             await SpaceConnector.clientV2.dashboard.domainDashboard.update({
-                ...state.apiParam,
-                name: state.name,
+                name: dashboardDetailState.name,
+                labels: dashboardDetailState.labels,
+                settings: dashboardDetailState.settings,
+                layouts: [dashboardDetailState.dashboardWidgetInfoList],
+                variables: dashboardDetailState.variables,
+                variables_schema: dashboardDetailState.variablesSchema,
+                tags: { created_by: store.state.user.userId },
```

Comment:
Suggestion: What do you think about using a separate variable and the spread operator here to reduce this repetitive part?

Reply:
I applied it :) thank you.